src/components/SearchForm/_search-form.scss

```diff
@@ -7,13 +7,13 @@
 // ----------------------------------
 
 .search-form {
-  padding-bottom: calc(4 * var(--space-unit));
-  padding-top: calc(4 * var(--space-unit));
+  margin-bottom: calc(4 * var(--space-unit));
+  margin-top: calc(4 * var(--space-unit));
```

Comment:
Could we use new spacings here? (`var(--space-xl)`)?

Reply:
Good shout